eng/pipelines/templates/steps/generate-doc.yml

```diff
@@ -0,0 +1,17 @@
+steps:
+  - script: |
+      npm i -g typedoc typescript
```

Comment:
Do we need to pin to a particular version here or is floating OK?

Reply:
latest is fine

Reply:
maybe i'll pin both the the current latest

Reply:
pinning typedoc to 0.6.x